package.json

```diff
@@ -18,8 +22,9 @@
     "nodemon": "^2.0.4"
   },
   "scripts": {
-    "start": "node index.js",
-    "dev": "env-cmd -f .env nodemon index.js",
+    "start": "node dist",
```

Comment:
We have auto deployments for Vacabot app using heroku.
And there we have configured `npm start` as the running script.

Reply:
Which means, this dist has to be created before running

Reply:
@mukarramali Can you Check once i think heroku builds automatically before deployments.
https://devcenter.heroku.com/changelog-items/1557

Reply:
that's great.
But we don't have build script in package.json.
You wanna add that?

Reply:
@mukarramali In the PR i have already added the build script. PR is good to go.

Reply:
Turns out babel didn't pick the environment variables.
To test it in your machine, you can first creat .env file as told in README. And then do build and start.
It should through an error as expected env variables aren't present.